distribution/src/config/jvm.options

```diff
@@ -37,6 +37,14 @@
 -XX:CMSInitiatingOccupancyFraction=75
 -XX:+UseCMSInitiatingOccupancyOnly
 
+## G1GC Configuration
+# NOTE: We only support G1GC on JDK version 10 or later.
```

Comment:
To me this read as, the only GC supported on JDK>=10 is G1GC (e.g. CMS not supported). Then next line reads, to "use G1GC uncomment the lines below".

Would it be better to rephrase as:

G1GC is only supported on JDK version 10 or later.

Reply:
Good idea, I've made that change, thanks!